src/components/button-toggle/button-toggle.component.tsx

```diff
@@ -107,8 +107,8 @@ export const ButtonToggle = ({
     isInGroup,
     firstButton,
     childButtonCallbackRef,
+    variant,
```

Comment:
This hasn't been resolved

Reply:
This will get the default value from the context as it's defaulted in the group component

Reply:
This only gets the default value from context if its wrapped in a `ButtonToggleGroup`, if its being used independently, or not wrapped it could cause issues. Looks like this is actually causing the cypress tests which Debra removed to fail, when its added they pass